hpx/parallel/executors/distribution_policy_executor.hpp

```diff
@@ -60,53 +72,54 @@ namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v3)
                 "distribution policy type");
 
         // apply_execute implementations
-        template <typename F>
+        template <typename F, typename ... Ts>
         typename std::enable_if<
             !hpx::traits::is_action<F>::value
         >::type
-        apply_execute(F && f) const
+        apply_execute_impl(F && f, Ts && ... ts) const
         {
             typedef components::server::invoke_function_action<F> action_type;
-            policy_.template apply<action_type>(
-                threads::thread_priority_default, std::forward<F>(f));
+            policy_.template apply<action_type>(threads::thread_priority_default,
+                std::forward<F>(f), std::forward<Ts>(ts)...);
```

Comment:
Well, nothing has changed relatively to the implementation of distribution_policies. I'd be happy to discuss this on a separate PR.